source/framework/vx_tensor.c

```diff
@@ -52,11 +52,16 @@ static vx_status isTensorSwappable(vx_reference input, vx_reference output)
     {
         tivx_obj_desc_tensor_t * ip_obj_desc = (tivx_obj_desc_tensor_t *)input->obj_desc;
         tivx_obj_desc_tensor_t * op_obj_desc = (tivx_obj_desc_tensor_t *)output->obj_desc;
+#ifdef LDRA_UNTESTABLE_CODE
+/* vxCreateTensorFromHandle does not exist in the current TI implementation 
+   thus this piece of code cannot be reached. We can keep this check for future immpelentation.
+ */         
         if (ip_obj_desc->mem_size != op_obj_desc->mem_size)
         {
            status = (vx_status)VX_ERROR_NOT_SUPPORTED;
        }
     }
+#endif
```

Comment:
Will this endif need to be moved up one line?

Reply:
yes moved up